src/javascript/binary/base/page.js

```diff
@@ -101,6 +113,11 @@ const Page = (() => {
         Menu.onUnload();
     };
 
+    const removeIcoBanner = () => {
+        document.getElementById('ico_banner').classList.add('invisible');
+        sessionStorage.setItem('icoBanner', true);
```

Comment:
maybe we can rename `icoBanner` to something like `hide_ico_banner`? to make it more clear what it's being set for, right now it seems the opposite of what it is

Reply:
Noted, I'll rename the session key.